oracle/types/market.go

```diff
@@ -64,3 +64,15 @@ func (tpt *TickersToProviderTickers) ToProviderTickers() []ProviderTicker {
 
 	return providerTickers
 }
+
+// MustGetProviderTicker returns the provider ticker for the given x/marketmap ticker.
+// This function is mostly used for testing.
+func (tpt *TickersToProviderTickers) MustGetProviderTicker(ticker mmtypes.Ticker) ProviderTicker {
```

Comment:
seems like this should just not be a pointer receiver function

Reply:
Is there a reason why?